.golangci.yml

```diff
@@ -1,47 +1,10 @@
-# https://golangci-lint.run/usage/configuration/
+version: "2"
 run:
-  timeout: 10m
```

Comment:
v1 had a default timeout of 1m so we had to bump it up since our linting would sometimes take a while. V2 has no timeout by default, so this is no longer an issue. You can configure the tiemout via a parameter when running `golangci-lint` now